examples/destination/java/src/main/java/destination/DestinationServiceImpl.java

```diff
@@ -48,26 +49,32 @@ public void configurationForm(ConfigurationFormRequest request, StreamObserver<C
 
     @Override
     public void capabilities(CapabilitiesRequest request, StreamObserver<CapabilitiesResponse> responseObserver) {
+        final int million = 1_000_000;
```

Comment:
This variable name is not very descriptive. We should create different ones for the different numeric max values we are setting, the variable name should explain what it is

Reply:
changed